config/jobs/kubernetes/community/community-presubmit.yaml

```diff
@@ -2,21 +2,17 @@ presubmits:
   kubernetes/community:
   - name: pull-community-verify
     branches:
-    - master
+    - ^master$
     always_run: true
-    labels:
-      preset-service-account: "true"
+    decorate: true
+    path_alias: k8s.io/community
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
-        args:
-        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-        - "--root=/go/src"
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - --scenario=execute
-        - --
-        - make
-        - verify
+      - image: golang:1.11
+        command:
+        - sh
+        - "-c"
+        - "export PATH=$PATH:$GOPATH/bin && make verify"
```

Comment:
Ref: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-pm/enhancements-presubmit.yaml

(earlier error was https://github.com/kubernetes/test-infra/pull/13126#issuecomment-504610676)

Reply:
@krzyzacy do you remember why we didn't add this to the PATH in general?

Reply:
If we are swapping the image out here for the upstream go image, then `$GOPATH/bin` is already in the $PATH